app/navigation/Navigation.scala

```diff
@@ -73,7 +75,9 @@ class Navigation @Inject()(dateTimeHelper: DateTimeHelper,
     AppealStartPage -> ((_, _, _) => routes.AppealStartController.onPageLoad()),
     OtherRelevantInformationPage -> ((_, _, _) => routes.OtherReasonController.onPageLoadForUploadEvidence(NormalMode)),
     CancelVATRegistrationPage -> ((answer, _, extraData) => routingForCancelVATRegistrationPage(answer, extraData)),
-    OtherPenaltiesForPeriodPage -> ((_, _, _) => routes.AppealStartController.onPageLoad())
+    OtherPenaltiesForPeriodPage -> ((_, _, _) => routes.AppealStartController.onPageLoad()),
+    UploadFirstDocumentPage -> ((_, _, _) => routes.OtherReasonController.onPageLoadUploadList(NormalMode)),
+    UploadAnotherDocumentPage -> ((_,_,_) => routes.OtherReasonController.onPageLoadUploadList(NormalMode))
```

Comment:
Not that this is an option yet, but you may want to consider adding a page for where to go from the 'You have uploaded X files' i.e. adding UploadListPage and then adding some functionality that would route to the making a late appeal or CYA page when the user says no or to the add another page if the user says yes to adding another document